src/explorer/Metrics/ExploreMetric.cs

```diff
@@ -6,6 +6,7 @@ public interface ExploreMetric
     {
         public string Name { get; }
 
+        [JsonPropertyName("value")]
```

Comment:
Maybe this is also the time to add a shape related test? After all we had the right shape in the past, then suddenly didn't. Let's guard against this regression happening again.

Reply:
👍 it's on our list (#170). Now that the api is stabilized + versioned we'll prioritize this.